### ISSUE TITLE
Missing `aria-invalid` selector

### ISSUE THREAD

**Issue by hckhanh** [tailwindlabs/tailwindcss#10127]
I am using the latest version of Tailwind and I think `aria-invalid` is missing

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-invalid

**Comment by adamwathan** [comment]
Hey! As per the docs we've only included the most common boolean attributes out of the box. Anything else you need you can add via your own configuration file or by using an arbitrary value:

https://tailwindcss.com/docs/hover-focus-and-other-states#aria-states

**Comment by msalahz** [comment]
I fixed this by installing https://www.npmjs.com/package/tailwindcss-aria-attributes
